CONTRIBUTING.md

```diff
@@ -141,6 +144,57 @@ to email the patch to <gap@gap-system.org>.  You can refer to
 [this tutorial](https://burzalodowa.wordpress.com/2013/10/05/how-to-send-patches-with-git-send-email/)
 on how to do this.
 
+## The reviewing process
+
+Before a change can be incorporated into the code base, our setup requires
+each pull request (PR) to be
+[reviewed](https://help.github.com/articles/about-pull-request-reviews/)
+by at least one GAP developer (i.e. anybody with write access to the
+[gap repository](https://github.com/gap-system/gap/)).
+
+If you don't have write access, you are nevertheless very welcome to submit
+code reviews! Search the
+[pull requests](https://github.com/gap-system/gap/pulls)
+for a PR that you would like to review. Once you have chosen a PR
+(e.g. [PR 2261](https://github.com/gap-system/gap/pull/2261)),
+open the `Files changed` tab
+(e.g. [this](https://github.com/gap-system/gap/pull/2261/files))
+and add your comments or click the green `Review changes` button in the top
+right.
+
+You can use the lists below as checklists for how to write your review.
+Please be careful to criticize constructively and not use dismissive language
+(see e.g. Brian Lee's section on `Rewording Feedback` in his
+[blog post](https://medium.com/unpacking-trunk-club/designing-awesome-code-reviews-5a0d9cd867e3)).
+
+
+### Before you dive into the code (taken from [here](https://lornajane.net/posts/2015/code-reviews-before-you-even-run-the-code))
```

Comment:
I'd move the link out of the title. E.g. write this as first sentence of this section:
> This section is based on [this article](...).